katherine95/ireporter-reactjs
# #165164504 Consume login endpoint.

#### What does this PR do?
Consume login endpoint.
#### Description of Task to be completed?
Fetch the login endpoint and implement a user interface to use to consume it.
#### How should this be manually tested?
Enter username(email) and password and click on the login button. This should log the user token generated.
#### What are the relevant pivotal tracker stories?
[#165164504](https://www.pivotaltracker.com/story/show/165164504)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/17095461/55808398-aba4d980-5aec-11e9-9983-0aab48469363.png)
